package models;

public class Course {

    private int course_id;
    private String course_name;
    private int credits;

    public Course(int course_id, String course_name, int credits) {
        this.course_id = course_id;
        this.course_name = course_name;
        this.credits = credits;
    }

    public int getCourse_id() {
        return course_id;
    }

    public String getCourse_name() {
        return course_name;
    }

    public int getCredits() {
        return credits;
    }
}
